frontend/src/lib/components/Tooltip.svelte

```diff
@@ -4,7 +4,7 @@
 	import Popover from './Popover.svelte'
 </script>
 
-<Popover notClickable>
+<Popover notClickable class="flex">
```

Comment:
<img width="116" alt="Screenshot 2023-01-06 at 07 23 37" src="https://user-images.githubusercontent.com/456655/211012060-f3ccb7d5-c4c5-4984-ae2b-c684494d880d.png">

I added this class to align the tooltip properly